### PR TITLE
[Sources] Fix Snowflake Failed Task

### PR DESCRIPTION
**Pull request by vellames**
### <samp>🤖 Generated by Copilot at c01d657</samp>

This pull request updates the snowflake package and several actions and sources that depend on it. It adds a new feature to the insert-multiple-rows action, fixes a bug in the insert-row action, and improves the accuracy and reliability of the failed-task-in-schema source. It also updates the versions and SQL queries of other sources to reflect the latest changes in the snowflake API.

### <samp>🤖 Generated by Copilot at c01d657</samp>

> _Snowflake package changed_
> _Actions and sources updated_
> _A fresh winter code_

### <samp>🤖 Generated by Copilot at c01d657</samp>

*  Bump the version of the snowflake package to 0.6.1 to reflect the changes in the actions and sources ([link](https://github.com/PipedreamHQ/pipedream/pull/6410/files?diff=unified&w=0#diff-9f1a179ed7be32b74e2e4b5d5b1158fc619865e5629751bc262062e95ca7760bL3-R3))
*  Add a new feature to the insert-multiple-rows action to allow inserting multiple rows at once, and bump its version to 0.0.3 ([link](https://github.com/PipedreamHQ/pipedream/pull/6410/files?diff=unified&w=0#diff-5dcb6e07b872e06c94ec09b23afa2f80f797df2ce61ae0140f2fdd62051dca9aL9-R9))
*  Fix a bug in the insert-row action that used the wrong table name in the SQL statement, and bump its version to 1.0.2 ([link](https://github.com/PipedreamHQ/pipedream/pull/6410/files?diff=unified&w=0#diff-0d457a538cafb0b75c2d2fb213cd68e2db88fcbae164393b487e00cf1d348cabL8-R8))
*  Improve the accuracy and reliability of the failed-task-in-schema source by using the COMPLETED_TIME column instead of the QUERY_START_TIME column in the SQL query, and change the event id from the queryId to the runId, and bump its version to 0.1.0 ([link](https://github.com/PipedreamHQ/pipedream/pull/6410/files?diff=unified&w=0#diff-867dee6344b167ef75ef90969a4511ccfd1cdc2bbc9579d7bcb6e9fe2df75536L177-R189), [link](https://github.com/PipedreamHQ/pipedream/pull/6410/files?diff=unified&w=0#diff-8463d90b1fce2b0a62c9996d446a3b269e680bc0598a76accdadd3ea59d268d2L181-R181), [link](https://github.com/PipedreamHQ/pipedream/pull/6410/files?diff=unified&w=0#diff-46d7a50b0508048458f0aead203cdcf1107b097999cb8e0d063faccd2e98ac3bL35-R35))
*  Update the versions of the other sources to 0.0.4 or 0.0.3 to reflect the dependency update of the snowflake package ([link](https://github.com/PipedreamHQ/pipedream/pull/6410/files?diff=unified&w=0#diff-ac245c12a952fbef3c8c416bbdfb2d521b99721145a089cbde3fe4efb825da2fL20-R20), [link](https://github.com/PipedreamHQ/pipedream/pull/6410/files?diff=unified&w=0#diff-3ddbe7bea9e4ecab5889cbe3f40d2d7337e0dee42a0b7dbdf8a734299f4a73f5L10-R10), [link](https://github.com/PipedreamHQ/pipedream/pull/6410/files?diff=unified&w=0#diff-2fa82442093eb4863a18b6984ae1c52f0d4a24d67326fa15d002d18816d9f31cL9-R9), [link](https://github.com/PipedreamHQ/pipedream/pull/6410/files?diff=unified&w=0#diff-2b0d9292f31caec05d5fba5f5586a3bb616076afadf2af619c878b869bd4a6d4L9-R9), [link](https://github.com/PipedreamHQ/pipedream/pull/6410/files?diff=unified&w=0#diff-a547f7581b5437077f6235a72c6b018f42ffed1019ca24e3951b173755a2ff87L10-R10), [link](https://github.com/PipedreamHQ/pipedream/pull/6410/files?diff=unified&w=0#diff-f738b23c10442b3b965f2377fcc88fd18d679aea39da15b3bf6c611e0196d2bbL10-R10), [link](https://github.com/PipedreamHQ/pipedream/pull/6410/files?diff=unified&w=0#diff-7ab25cafd22b71d453819949e30d615a936540e62e2dc463a329417fc2f06767L9-R9), [link](https://github.com/PipedreamHQ/pipedream/pull/6410/files?diff=unified&w=0#diff-728a871fc8a509ef8491ed6aea54840f9e65bcd808b011766eeeda90e4aa35f4L11-R11))
*  Use the CREATED column instead of the deprecated CREATED_ON column in the SQL query for the new-database source ([link](https://github.com/PipedreamHQ/pipedream/pull/6410/files?diff=unified&w=0#diff-3ddbe7bea9e4ecab5889cbe3f40d2d7337e0dee42a0b7dbdf8a734299f4a73f5L10-R10))

### CODE REVIEW

File: components/snowflake/sources/new-database/new-database.mjs [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default {
   key: "snowflake-new-database",
   name: "New Database",
   description: "Emit new event when a database is created",
-  version: "0.0.2",
+  version: "0.0.4",
```
Comment (GTFalcao) [inline components/snowflake/sources/new-database/new-database.mjs:10]:
```suggestion
  version: "0.0.3",
```

File: components/snowflake/sources/failed-task-in-schema/failed-task-in-schema.mjs [hunk 2]
```diff
@@ -32,7 +32,7 @@ export default {
   // eslint-disable-next-line
   name: "Failed Task in Schema",
   description: "Emit new events when a task fails in a database schema",
-  version: "0.0.4",
+  version: "0.1.0",
```
Comment (GTFalcao) [inline components/snowflake/sources/failed-task-in-schema/failed-task-in-schema.mjs:35]:
```suggestion
  version: "0.0.5",
```

**Review by GTFalcao — COMMENTED** [review]
Hi @vellames , thanks for the fixes, I think some version numbers should be updated for new `patch` versions, but I'm moving this forward to speed things up.

**Review by GTFalcao — APPROVED** [review]

**Comment by vellames** [comment]
For QA context https://pipedream-users.slack.com/archives/CPTJYRY5A/p1683174526368959

**Comment by vunguyenhung** [comment]
Hello, I can see that there's event emitted for every failed tasks, even when they're emitted at different time
image
image
image


Ready for release!

**Pull request merged**